### DISCUSSION TITLE
I used the STM32F103RCT6 to drive the P4 outdoor screen RGB full color scan mode to 10 scans but it doesn't seem to work

### DISCUSSION THREAD

**Discussion by ShiPC123** [board707/DMD_STM32#103] [Q&A]
![1](https://github.com/board707/DMD_STM32/assets/132763025/a1d624b2-6b29-4ef7-a10f-662d0b3b31e5)
![2](https://github.com/board707/DMD_STM32/assets/132763025/16d4655c-a8f4-467c-ae18-ddd720358bb3)
![3](https://github.com/board707/DMD_STM32/assets/132763025/2f98f591-7424-4b4c-9f7c-3765c62ed28b)

**Comment by board707** [comment]
Hi
What is your panel's pixel dimensions and scan factor? Please take a good resolution picture of panel's rear side.
Please show the headings for all types the chips on the panel.

**Comment by ShiPC123** [comment]
HI Some photos and board connection I put in the issues specific board interface definition figure
![4ae99b7eae21f791fe902402b1bf1513](https://github.com/board707/DMD_STM32/assets/132763025/b22475f5-4f7e-4c15-84d2-83ad73bf197f)
https://github.com/board707/DMD_STM32/issues/104

**Comment by ShiPC123** [comment]
The pixel of the dot screen is 80X40 1/10 scan  The circuit diagram is shown
<img width="576" alt="企业微信截图_17146521847920" src="https://github.com/board707/DMD_STM32/assets/132763025/4cfdfecc-1fa8-4ab1-8ce8-332e6e0484f1">
of the dot screen is 80X40 1/10 scan 
<img width="490" alt="企业微信截图_17146522105507" src="https://github.com/board707/DMD_STM32/assets/132763025/cf19af26-2279-43ee-8735-60e3487695a2">

**Comment by board707** [comment]
Continued in #104
